pixeltable/globals.py

```diff
@@ -560,7 +560,7 @@ def list_tables(dir_path: str = '', recursive: bool = True) -> list[str]:
 
 
 def create_dir(
-    path: str, if_exists: Literal['error', 'ignore', 'replace', 'replace_force'] = 'error'
+    path: str, if_exists: Literal['error', 'ignore', 'replace', 'replace_force'] = 'error', create_parents: bool = False
```

Comment:
In the Python API this is called simply `parents`, not `create_parents`. We might want to consider calling it simply `parents` for consistency. This kind of makes sense, since the `create_` is implied by the method name.

See:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir